critiquebrainz/frontend/templates/macros.html

```diff
@@ -35,3 +35,19 @@
     <abbr title="{{ _('Submit and delete your votes.') }}">{{ _('Review votes') }}</abbr>
   {% endif %}
 {% endmacro %}
+
+{% macro coverart(entity_id, entity_type, attributes='') %}
+  {# Note: when passing in `attributes`, don't quote attribute values; they will be added automatically #}
+  {% if entity_type == 'event' %}
+    <img src="/static/img/event-placeholder.png" {{ attributes }} />
```

Comment:
@gentlecat I must have not realized this after I switched to using a SVG from a PNG; thanks! Resolved in cea06b1